suite/tests/client-interface/low_on_memory.dll.c

```diff
@@ -138,6 +145,8 @@ exit_event(void)
     if (!is_clear)
         dr_fprintf(STDERR, "was not cleared!\n");
 
+    free_nodes();
```

Comment:
This is not in the PR description: is this fixing a memory leak? Please update the description.